src/builder-git.c

```diff
@@ -476,7 +476,8 @@ builder_git_mirror_repo (const char     *repo_location,
   gboolean update = (flags & FLATPAK_GIT_MIRROR_FLAGS_UPDATE) != 0;
   gboolean disable_fsck = (flags & FLATPAK_GIT_MIRROR_FLAGS_DISABLE_FSCK) != 0;
 
-  gboolean disable_lfs = !git_lfs_is_available ();
+  gboolean disable_lfs = (flags & FLATPAK_GIT_MIRROR_FLAGS_DISABLE_LFS) != 0 ||
+                         !git_lfs_is_available ();
```

Comment:
Given that you check both things everywhere, this is a good candidate to pull in its own function. `builder_git_use_lfs`?

Reply:
Makes sense.